File: src/main/java/com/github/liberatemetumortis/Programmer.java
package com.github.liberatemetumortis;

public class Programmer extends Person {
    private String language;

    public String getLanguage() {
        return language;
    }

    public void setLanguage(String language) {
        this.language = language;
    }

    public Programmer(int age, String name, String language) {
        super(age, name);
        this.language = language;
    }

    @Override
    public String toString() {
        return "Programmer [language=" + language + ", getAge()=" + getAge() + ", getName()=" + getName() + "]";
    }

    @Override
    public void describeYourSelf() {
        System.out.println("Ben bir programcıyım.");
    }


}

File: src/main/java/com/github/liberatemetumortis/Programmer.kt
package com.github.liberatemetumortis

class Programmer(age: Int, name: String?, var language: String) : Person(age, name!!) {

    override fun toString(): String {
        return "Programmer [language=" + language + ", getAge()=" + getAge() + ", getName()=" + name + "]"
    }

    override fun describeYourSelf() {
        println("Ben bir programcıyım.")
    }
}